add_to_app/multiple_flutters/multiple_flutters_android/app/src/main/AndroidManifest.xml

```diff
@@ -37,6 +37,15 @@
             android:hardwareAccelerated="true"
             android:windowSoftInputMode="adjustResize"
             />
+        <!-- Whether leave or clean up the VM after the last shell shuts down. If your want to let
+             your app destroy the last shell and re-create shells more quickly, set it to true,
+             otherwise if you want to clean up the memory of the leak VM, set it to false. This
+             setting should works after the Flutter engine supports custom switch. View this link:
```

Comment:
When will this custom switch in engine make it to stable?

Reply:
after 2.10, which is the next stable.

Reply:
Done. Add a 2.10 stable version comment. cc @blasten @domesticmouse  Thanks 😄